File: unit-test-practice-8/src/test/java/dojo/SnakeCaseTransformerTest.java
package dojo;

import static org.assertj.core.api.Assertions.assertThat;
import static org.assertj.core.api.Assertions.assertThatThrownBy;

import org.junit.Test;

public class SnakeCaseTransformerTest {

	private SnakeCaseTransformer transformer = new SnakeCaseTransformer();
	
	@Test
	public void sample_test() throws Exception {
		String result = transformer.transform("");
		assertThat(4).isEqualTo(4);
	}
}

File: unit-test-practice-8/src/test/csharp/dojo.Tests/SnakeCaseTransformerTest.cs
namespace dojo.Tests;

using System;
using NUnit.Framework;

public class SnakeCaseTransformerTest
{
    private readonly SnakeCaseTransformer transformer = new SnakeCaseTransformer();

    [Test]
    public void SampleTest()
    {
        Assert.AreEqual(4, 4);
    }
}

